gnat2goto/driver/tree_walk.adb

```diff
@@ -3729,6 +3780,54 @@ package body Tree_Walk is
 
    end Do_Unconstrained_Array_Definition;
 
+   -------------------------
+   -- Do_Withed_Unit_Spec --
+   -------------------------
+
+   procedure Do_Withed_Unit_Spec (N : Node_Id) is
+      Unit_Name : constant String := Get_Name_String (Get_Unit_Name (N));
+   begin
+      if Defining_Entity (N) = Stand.Standard_Standard or else
+        Unit_Name = "system%s"
+      then
+         null;
+         --   At the moment Standard or System are not processed - to be done
+      else
+
+         case Nkind (N) is
+            when N_Subprogram_Body =>
+               if Acts_As_Spec (N) then
+                  --  The unit is a withed library unit which subprogram body
```

Comment:
suggest add summary line at top (similar to this)
--  withed library unit that has no separate declaration
(this gives the reader what is common among the scenario discussed below)

Reply:
Done.